lessons/Data-Science/histograms/langs/en-us/index.adoc

```diff
@@ -103,7 +103,7 @@ To build a histogram:
   * If our values ranged from 3 to 53 we might mark our x-axis from 0 to 60 and divide it into bins of width 10.
   * If they range from 22 to 41 we might mark our x-axis from 20 to 45 and divide it into bins of width 5.
 - Put each value in our dataset into the bin where it belongs, and then count how many values fall in each bin.
-- The count of the values in each bin determines the height of the bars on our y-axis.
+- The number of the values in each bin determines the height of the bars on our y-axis.
```

Comment:
remove the second "the" so that it reads

The number of values in each bin determines the height of the bars on our y-axis.

Reply:
done